ios/MullvadREST/RESTAuthorization.swift

```diff
@@ -37,6 +38,10 @@ extension REST {
                 })
             }
         }
+
+        func invalidateToken(for accountNumber: String) {
```

Comment:
AccessTokenProvider is bound by accountNumber so technically you don't have to accept `accountNumber` as input, you could just use `self.accountNumber`